Sources/SwiftBuild/SWBBuildParameters.swift

```diff
@@ -36,21 +36,131 @@ public struct SWBBuildParameters: Codable, Sendable {
 
 /// Refer to `SWBProtocol.RunDestinationInfo`
 public struct SWBRunDestinationInfo: Codable, Sendable {
-    public var platform: String
-    public var sdk: String
-    public var sdkVariant: String?
+    public struct SWBBuildTarget: Codable, Sendable {
+        var _internalBuildTarget: InternalBuildTarget
+
+        public static func toolchainSDK(platform: String, sdk: String, sdkVariant: String?) -> SWBBuildTarget {
+            SWBBuildTarget(_internalBuildTarget: .toolchainSDK(platform: platform, sdk: sdk, sdkVariant: sdkVariant))
+        }
+
+        public static func swiftSDK(sdkManifestPath: String, triple: String) -> SWBBuildTarget {
+            SWBBuildTarget(_internalBuildTarget: .swiftSDK(sdkManifestPath: sdkManifestPath, triple: triple))
+        }
+
+        public func asToolchainSDK() -> (platform: String, sdk: String, sdkVariant: String?)? {
```

Comment:
nit: this and asSwiftSDK aren't used anymore and could be removed, unless we expect to need to call them.